content/en/content-management/summaries.md

```diff
@@ -21,25 +21,26 @@ With the use of the `.Summary` [page variable][pagevariables], Hugo generates su
 
 ## Summary Splitting Options
 
-* Hugo-defined Summary Split
-* User-defined Summary Split
+* Automatic Summary Split
+* Manual Summary Split
 
 It is natural to accompany the summary with links to the original content, and a common design pattern is to see this link in the form of a "Read More ..." button. See the `.RelPermalink`, `.Permalink`, and `.Truncated` [page variables][pagevariables].
 
-### Hugo-defined: Automatic Summary Splitting
+### Automatic Summary Splitting
 
-By default, Hugo automatically takes the first 70 words of your content as its summary and stores it into the `.Summary` page variable for use in your templates. Taking the Hugo-defined approach to summaries may save time, but it has pros and cons:
+By default, Hugo automatically takes the first 70 words of your content as its summary and stores it into the `.Summary` page variable for use in your templates. You may customize the summary length by setting `summaryLength` in your [site configuration](/getting-started/configuration/).
 
-* **Pros:** Automatic, no additional work on your part.
-* **Cons:** All HTML tags are stripped from the summary, and the first 70 words, whether they belong to a heading or to different paragraphs, are all put into one paragraph.
+{{% note %}}
+You can customize how HTML tags in the summary are loaded using funcitons such as `plainify` and `safeHTML`.
+{{% /note %}}
 
 {{% note %}}
-The Hugo-defined summaries are set to use word count calculated by splitting the text by one or more consecutive white space characters. If you are creating content in a `CJK` language and want to use Hugo's automatic summary splitting, set `hasCJKLanguage` to `true` in you [site configuration](/getting-started/configuration/).
+The Hugo-defined summaries are set to use word count calculated by splitting the text by one or more consecutive whitespace characters. If you are creating content in a `CJK` language and want to use Hugo's automatic summary splitting, set `hasCJKLanguage` to `true` in your [site configuration](/getting-started/configuration/).
 {{% /note %}}
 
-### User-defined: Manual Summary Splitting
+### Manual Summary Splitting
 
-Alternatively, you may add the <code>&#60;&#33;&#45;&#45;more&#45;&#45;&#62;</code> summary divider where you want to split the article. For [org content][org], use `# more` where you want to split the article. Content that comes before the summary divider will be used as that content's summary and stored in the `.Summary` page variable with all HTML formatting intact.
+In addition, you may add the <code>&#60;&#33;&#45;&#45;more&#45;&#45;&#62;</code> summary divider where you want to split the article. For [org content][org], use `# more` where you want to split the article. Content that comes before the summary divider will be used as that content's summary and stored in the `.Summary` page variable with all HTML formatting intact.
```

Comment:
This is an alternative and not an additional option, correct? As in, you can't do _both_ the "automatic" and "manual"  summaries for a given piece of content. Is this a gratuitous correction, @brennanangel?

Reply:
Thank you for the feedback @rdwatters! 
I believe it's an important clarification because in my testing you are able to do both.   

You can set the default summary length at a global level and use the manual summary splitting for individual articles.   If you couldn’t do both automatic and manual, “Alternatively,” would be more clear. 

I was hoping this change would clarify that both are an option. 

Do you think it's clear that a user can do both with "Alternatively"? If so, yes, this line may be an unnecessary change.